fields.yml

```diff
@@ -381,35 +381,55 @@
             Unique ID to describe the event.
           example: 8a4f500d
     
+        - name: kind
+          level: core
+          type: keyword
+          description: >
+            The kind of the event.
+    
+            This gives information about what type of information the event
+            contains, without being specific to the contents of the event.  Examples
+            are `event`, `state`, `alarm`.
+          example: state
+    
         - name: category
           level: core
           type: keyword
           description: >
             Event category.
     
-            This can be a user defined category.
-          example: metrics
+            This contains high-level information about the contents of the event. It
+            is more generic than `event.action`, in the sense that typically a
+            category contains multiple actions.
+          example: user-management
```

Comment:
The plan with this field is to have a predetermined set of categories (a few tens of them). Having the field now in ECS sets us up for a sort of breaking change when we come up with that list.

Therefore I think the description can remain as is, but we should add a warning telling people that they're using this field at their own risk, because it will soon be a field that should be populated with prescribed values.

It won't be the end of the world, though. It's not going to break ingestion or anything. It's just that Elastic solutions will expect this field to contain certain values, so if some sources populate this differently, they won't have the best experience...

Reply:
warning added.